task5/classes/Status.java

```diff
@@ -0,0 +1,13 @@
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet("/status")
+public class Status extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        response.getWriter().write("<p style='color:red'>Application Is Running</p>");
```

Comment:
The constants rule is universal, it is for all the literals and "magic numbers" anywhere they are used